entries/o/o2.pl.json

```diff
@@ -0,0 +1,14 @@
+{
+  "o2": {
+    "domain": "o2.pl",
+    "url": "https://www.o2.pl/",
+    "tfa": [
+      "totp",
+      "custom-software"
+    ],
+    "documentation": "https://pomoc.o2.pl/1login/jak-wlaczyc-i-wylaczyc-weryfikacje-dwuetapowa",
```

Comment:
Are o2 and WP the same service?
I see they use the same documentation page.

Reply:
Both have the same owner, but they are different services.